web-console/1-homepage/catalog.md

```diff
@@ -1,51 +1,47 @@
-*******************************
-Things you may want to include in this file
-Browse
-filter
+# Catalog
 
-*******************************
+- The Catalog section is found at the bottom of the web console homepage.
+- The Catalog contains provisionable items and provides the means to browse, sort, and filter them.
 
 
-# Template
+## Browse Catalog
 
-- This is a description of the section and where it is located/how to get to it
-- This is a description of what this section is for
+![catalog](img/catalog-01.png)
+- Initially, upon arriving on the page, the `All Categories` section is expanded and the resources are sorted alphabetically by name.
 
+#### Implementation Details
+  - Administrators should have the ability to determine which items are available to users;      
+    - Disabled items should not appear.
+    - Any tabs that do not contain any items should not be displayed.
+  - Initially, category labels will be limited to those that we define, potentially including an “Other category” for customers to place resources that don’t fit in an existing category. Eventually, customers should have the capability to add their own category headers.
 
-## Use Case 1 (e.g. Browse for service)
+![catalog](img/catalog-02.png)
+- Any tabs that have subcategories of items will have those subcategories displayed as tiles.
+- These tabs should also contain an a `All <category>` tile that contains items from all subcategories.
 
-![template](img/template-01.png)
-- description of what is happening in this wireframe
 
-#### Implementation Details
-  - Note PatternFly design patterns
-  - Note icon references
-
-![template](img/template-02.png)
-- description of what is happening in this wireframe
 
 #### Implementation Details
-  - Note PatternFly design patterns
-  - Note icon references
+  - Any subcategories that do not contain items should not be displayed.
+  - If a tab has subcategory tiles, they should not be mixed with regular service/items. Loose items should be placed in an `Other` tile.
 
-#### Customer Feedback (if applicable)
-- Customer Feedback Received
-  - Overview of customer feedback received
-- Necessary Customer Feedback
-  - Questions to follow up with customers
+![catalog](img/catalog-03.png)
+- Clicking on a subcategory will open a scrollable panel that contains the services or other items
+-
 
 
-## Use Case 2
+## Filter Catalog Items
 
-![template](img/template-03.png)
-- description of what is happening in this wireframe
+![catalog](img/catalog-04.png)
+- Opening the filter dropdown enables the user to select a number of filters to apply to the current set of items
 
 #### Implementation Details
-  - Note PatternFly design patterns
-  - Note icon references
+  - Filters between categories are combined with an AND relationship
+  - Filters within a category are combined with an OR relationship
```

Comment:
@cshinn let's verify what is possible for MVP with Dave & Jeff and update appropriately.  We can always include mixing AND and OR post 3.6